package.json

```diff
@@ -15,7 +15,9 @@
     "posttest": "cross-env NODE_ENV=test npm run db-migrate:undo",
     "lint": "eslint --init --fix",
     "start-dev": "nodemon --exec babel-node --presets env app.js",
-    "db-migrate": "sequelize db:migrate",
+    "db-migrate": "sequelize db:migrate && npm run db-seed",
```

Comment:
should we be seeding anytime we run migrations?

Reply:
I thought it will be appropriate to always have an admin when we migrate data